javascript/ecmascript-2015/symbol/symbol-special-properties.md

```diff
@@ -31,42 +31,42 @@ Symbols behave differently to other primitives such as strings and numbers.
 Lets imagine for this section we declare the following object:
 
 ```javascript
-var obj = {}
-obj["company"] = "Enki"
-obj[Symbol()]="You wont see me";
+var obj = {};
+obj["company"] = "Enki";
+obj[Symbol()] = "You wont see me";
```

Comment:
I guess we should use "won't" instead of "wont" here

Reply:
for both instances of 'wont'?

Reply:
done